stable/sparkoperator/templates/spark-operator-serviceaccount.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.serviceAccounts.sparkoperator.create }}
+{{- if and (.Values.deployment.create) (.Values.serviceAccounts.sparkoperator.create) }}
```

Comment:
why is this conditioned on the creation of the deployment?
if someone installed the cluster-rbac at point A, namespaced-rbac and point B, and deployment at point C
it should be created at point B I think